tests/unit_tests/scheduler/test_openpbs_driver.py

```diff
@@ -327,3 +327,33 @@ async def test_that_qdel_will_retry_and_succeed(
     assert "TRIED" in Path(bin_path / "script_try").read_text(encoding="utf-8")
     assert "qdel executed" in Path(bin_path / "qdel_output").read_text(encoding="utf-8")
     assert error_msg in Path(bin_path / "qdel_error").read_text(encoding="utf-8")
+
+
+@pytest.mark.usefixtures("capturing_qsub")
+@pytest.mark.parametrize(
+    "queue_config_string_for_keep_qsub, expectedkeep",
+    [
+        ("", False),  # Driver default
```

Comment:
I like that the test, but essentially all strings that are not in `["1", "True", "TRUE", "T"]` will be False, which sort of makes all the False cases insignificant.

Reply:
Suggestions: either make explicit that `["1", "True", "TRUE", "T"]` and `["0", "False", "FALSE", "F"]` is false, or just make one case that is false?

Reply:
Any other False scenario will be caught by the config validation.